fansi/shared/src/test/scala/fansi/FansiTests.scala

```diff
@@ -170,16 +170,42 @@ object FansiTests extends TestSuite{
       }
     }
     def tabulate(all: Seq[fansi.Attr]) = {
-      all.map(attr => attr.toString + " " * (25 - attr.name.length))
+      all.map(attr => attr.toString + " " * (30 - attr.name.length))
          .grouped(3)
          .map(_.mkString)
          .mkString("\n")
     }
 
+    def square(all : Seq[fansi.Attr]) = {
+      all.map( attr => attr.escapeOpt.getOrElse("") + "#")
+      .grouped(32)
+      .map(_.mkString)
+      .mkString("\n")
```

Comment:
the above map had 3 space indent, I had reduced it to 2 as like the rest of the file.